src/user.js

```diff
@@ -0,0 +1,14 @@
+const { method } = require('./http');
+const mapValues = require('lodash/mapValues');
+
+
+const getPermissions = (options = {}) => ({
+  method: 'GET',
+  url: `/user/permissions/`,
```

Comment:
If we are going to be changing removing of permissions to be possible via `DELETE /permissions/<id>`, then we need to move the current `/permissions/` method that obtains a user's permissions from a token to somewhere else. I vote for `GET /user/permissions/`, but up for other suggestions.